safe/impact_functions/base.py

```diff
@@ -208,6 +234,26 @@ def actual_extent_crs(self):
         return self._actual_extent_crs
 
     @property
+    def map_canvas(self):
```

Comment:
> To be more consistent with other properties like requested_extent and actual_extent, I can use map_canvas_extent. Is-it ok ?

Do you mean that we will only store the extent not the whole map canvas object? If so, I agree with it :)